CHANGELOG.md

```diff
@@ -1,110 +1,166 @@
+## 4.0.0
```

Comment:
Full version changelog should contain all changes between last stable version so alll stuff that dev version includes